ark/two/delegate.py

```diff
@@ -0,0 +1,22 @@
+from ark.two.api import API
+
+
+class Delegate2(API):
+
+    def delegate(self, id):
+        return self.get(f'api/v2/delegates/{id}')
+
+    def delegates(self, parameters=None):
```

Comment:
Instead of accepting dictionary for parameters, why not specify which args/kwargs can user pass in?

Reply:
I wanted the API calls to be as flexible and clean as possible. Also if any changes are made in the future to API inputs it won't break the API call and then we can just focus on maintaining the docstrings